### PR TITLE
Update Nugets and fixes first issues after updates

### PR DESCRIPTION
**Pull request by DeadLineSS13**

### CODE REVIEW

File: .github/workflows/dotnetcore.yml [hunk 1]
```diff
@@ -52,4 +52,7 @@ jobs:
       - name: Run dotnet format
         uses: jfversluis/dotnet-format@v1.0.5
         with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          action: fix
+          only-changed-files: false
```
Comment (corp-0) [inline .github/workflows/dotnetcore.yml]: I would keep this one to `true`

Comment (DeadLineSS13) [reply .github/workflows/dotnetcore.yml]: Ok. Now do it

**Comment by corp-0** [comment]
can you remove the fixes on warnings? We should handle those in another pr

**Comment by DeadLineSS13** [comment]
> can you remove the fixes on warnings? We should handle those in another pr

This fixes after update Nugets Libraries. I can fixes CI in other PR

**Comment by corp-0** [comment]
solve the conflicts to get the new workflow and pass dotnet-format

**Comment by corp-0** [comment]
already done in #163

**Comment by corp-0** [comment]
ah no my bad, reopened.

**Review by corp-0 — DISMISSED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**